raiden/network/proxies/token_network_registry.py

```diff
@@ -89,7 +92,36 @@ def get_token_network(
 
         return address
 
-    def add_token(self, token_address: TokenAddress, given_block_identifier: BlockSpecification):
+    def add_token_with_limits(
```

Comment:
for functions `add_token_(with|without)_limits` are meant to say "(with|without)" limits call arguments... if you have better suggestions, i am open to them

Reply:
Name should be fine. Perhaps just add this extra node you mentioned in the comment about the limit in the argument to the docstring of this function and of the next one `add_token_without_limits`